core/test/functional/routes/api/upload_profile_image_spec.js

```diff
@@ -0,0 +1,147 @@
+var should = require('should'),
+    supertest = require('supertest'),
+    testUtils = require('../../../utils'),
+    path = require('path'),
+    fs = require('fs-extra'),
+    ghost = testUtils.startGhost,
+    config = require('../../../../../core/server/config'),
+    request;
+
+describe('Upload Profile Image API', function () {
```

Comment:
PR looks good in general 👍. I need to do a manual test tomorrow morning.

One thing we agreed on is avoiding to use many routing tests (see [discussion](https://github.com/TryGhost/Ghost/issues/9178#issuecomment-340726993)), because they are slow.
The rule is: 1 successful routing test, maybe 1 error case and for everything else (edge cases) unit tests.

We haven't had the time yet to refactor and reduce the routing/integration tests yet. But we want to in the near future :)

Also: you could put these tests into the `upload_spec` file - not sure why `upload_icon_spec` exists 🤔 These are all grouped under the name root route `/uploads/`.

Reply:
Thanks @kirrg001 , added the profile image routing test to `upload_spec` and removed `upload_profile_image_spec`